jwst/assign_wcs/niriss.py

```diff
@@ -138,7 +138,12 @@ def imaging_distortion(input_model, reference_files):
 
 
 exp_type2transform = {'nis_image': imaging,
-                      'nis_wfss': not_implemented_mode,#        ?? WFSS spec
+                      'nis_wfss':
                       'nis_soss': niriss_soss,
-                      'nis_ami': not_implemented_mode#        ?? imaging
+                      'nis_ami': not_implemented_mode,
+                      'nis_tacq': imaging,
+                      'nis_taconfirm': imaging,
+                      'nis_focus': imaging,
+                      'nis_dark': not_implemented_mode,
+                      'nis_lamp': not_implemented_mode,
```

Comment:
This list is complete. nis_wfss doesn't have an entry. It's not implemented yet, but will be in B7.1. I'm guessing that nis_ami would be assigned to imaging, but we should confirm with the NIRISS team.